app/javascript/components/CurriculaSearch.js

```diff
@@ -37,17 +37,28 @@ class CurriculaSearch extends React.Component {
             checkboxbools: []
```

Comment:
can you pre-select "Select all of the above"
that makes more sense as a user experience. just put it in the initial state i think